### PR TITLE
CWR service enabled to work after reboot. Clean up skip errors.

### PR DESCRIPTION
**Pull request by ktsakalozos**

### CODE REVIEW

File: lib/CIGateway.py [hunk 1]
```diff
@@ -27,11 +27,14 @@ def start(cls, jenkins_url, jenkins_user, jenkins_pass, user='jenkins'):
...
 
+        subprocess.check_call(['systemctl', 'enable', 'cwr-server'])
         subprocess.check_call(['systemctl', 'daemon-reload'])
         host.service_start('cwr-server')
 
     @classmethod
     def stop(cls):
+        subprocess.check_call(['systemctl', 'disable', 'cwr-server'])
+        subprocess.check_call(['systemctl', 'daemon-reload'])
         host.service_stop('cwr-server')
```
Comment (kwmonroe) [inline lib/CIGateway.py]: Similar to the above, let's do something like:

```python
charmhelpers.core.host.service_pause('cwr-server')
if charmhelpers.core.host.init_is_systemd():
    subprocess.check_call(['systemctl', 'daemon-reload'])
charmhelpers.core.host.service_stop('cwr-server')
```

File: lib/CIGateway.py [hunk 2]
```diff
@@ -27,11 +27,14 @@ def start(cls, jenkins_url, jenkins_user, jenkins_pass, user='jenkins'):
                    'charm_dir': hookenv.charm_dir()
                })
 
+        subprocess.check_call(['systemctl', 'enable', 'cwr-server'])
         subprocess.check_call(['systemctl', 'daemon-reload'])
         host.service_start('cwr-server')
```
Comment (kwmonroe) [inline lib/CIGateway.py]: Please use charmhelpers.core.host.service_x methods so we don't hate ourselves if we ever need to support something besides systemd:

http://bazaar.launchpad.net/~charm-helpers/charm-helpers/devel/view/head:/charmhelpers/core/host.py#L59

```python
charmhelpers.core.host.service_enable('cwr-server')
if charmhelpers.core.host.init_is_systemd():
    subprocess.check_call(['systemctl', 'daemon-reload'])
charmhelpers.core.host.service_start('cwr-server')
```

File: lib/CIGateway.py (hunk 1 above)
Comment (ktsakalozos) [inline lib/CIGateway.py]: And yes! Thanks

File: lib/CIGateway.py (hunk 2 above)
Comment (ktsakalozos) [inline lib/CIGateway.py]: Yes!

**Pull request merged**